composer.json

```diff
@@ -18,12 +18,11 @@
     "jakub-onderka/php-parallel-lint": "^1.0",
     "localheinz/composer-normalize": "~0.9.0",
     "phing/phing": "^2.16.0",
-    "spryker/kernel": "*",
     "squizlabs/php_codesniffer": "^3.3.1",
-    "spryker/code-sniffer": "dev-master",
     "slevomat/coding-standard": "^4.7.2",
     "consistence/coding-standard": "^3.5",
-    "phpstan/phpstan-strict-rules": "^0.10"
+    "phpstan/phpstan-strict-rules": "^0.10",
+    "spryker/code-sniffer": "*"
```

Comment:
You can add ordered true into the config below and then this will be auto-sorted